apache/couchdb-fauxton
# Allow users to toggle word wrapping on the doc editor

## Overview

Allow users to toggle word wrapping on the doc editor, which prevents horizontal scrolling on long lines.

Word wrap OFF
<img width="1006" alt="image" src="https://github.com/apache/couchdb-fauxton/assets/30349380/955e16ac-52d4-4445-b397-c00e7c9f3ad6">

Word wrap ON
<img width="1007" alt="image" src="https://github.com/apache/couchdb-fauxton/assets/30349380/24481a4b-6d11-4d6b-934c-4a27acb598bc">


## Testing recommendations

- Edit a document and use the new preferences button in the toolbar to switch on/off word wrapping.
- Reload the page and check that your previous selection is kept.

## GitHub issue number

n/a

## Related Pull Requests

n/a

## Checklist

- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/main/rebar.config.script) with the correct tag once a new Fauxton release is made
